src/main/java/JavaQuestionOne/Main.java

```diff
@@ -0,0 +1,168 @@
+                System.out.println(curObj.getName()+"--------"+curObj.getQuantity()+"--------"+curObj.getItemType() + "--------" + curObj.calculateFinalAmount(curObj.getPrice(), curObj.getQuantity()));
+                System.out.println("-------------------------------------------------------------------------------------------------------------------------------");
+            } else  {
+                ImportedItem curObj = (ImportedItem) obj.get(index);
+                System.out.println(curObj.getName()+"--------"+curObj.getQuantity()+"--------"+curObj.getItemType() + "--------" + curObj.calculateFinalAmount(curObj.getPrice(), curObj.getQuantity()));
+                System.out.println("-------------------------------------------------------------------------------------------------------------------------------");
+            }
+        }
+    }
+
+    public boolean fetchNameFromInput(HashMap<String, String> arguments) {
+        String val = arguments.get("-name");
+        if (val != null ) {
+            if (!val.trim().isEmpty()) {
+                return true;
+            }
+        }
+            return false;
+    }
+
+    public boolean fetchQuantityFromInput(HashMap<String, String> arguments) {
+        String val = arguments.get("-quantity");
+        if (val != null ) {
+            if (!val.trim().isEmpty()) {
+                return true;
+            }
+        }
+        return false;
+    }
+
+    public boolean fetchTypeFromInput(HashMap<String, String> arguments) {
+        String val = arguments.get("-type");
+        if (val != null ) {
+            if (!val.trim().isEmpty()) {
+                return true;
+            }
+        }
+        return false;
+
+    }
+
+    public boolean fetchPriceFromInput(HashMap<String, String> arguments) {
+        String val = arguments.get("-price");
+        if (val != null ) {
+            if (!val.trim().isEmpty()) {
+                return true;
+            }
+        }
+        return false;
+    }
+
+    public HashMap<String, String> createMap(String[] args) {
+        HashMap<String, String> arguments = new HashMap<String, String>();
+        for (int counter = 0; counter < args.length - 1; counter += 2) {
+            arguments.put(args[counter], args[counter + 1]);
+        }
+        return arguments;
+    }
+
+    public String[] askUserToContinue(ArrayList<Item> obj){
+        System.out.println("Enter -name<Item Name> -price<Item Price> -quantity<Item Quantity> -type<Item type> ");
+        sc.nextLine();
+        String inp = sc.nextLine();
+        return inp.split(" ");
+    }
+
+    public static void main(String[] args) {
+        Main mainObj = new Main();
+        ArrayList<Item> obj = new ArrayList<Item>();
+        try {
+            mainObj.fetchCLArgValues(args, obj, mainObj );
+        }catch(Exception e) {
+            mainObj.printDetails(obj);
+            System.out.println(e);
+        }
+    }
```

Comment:
keep the main method in a separate class, let that just be an entry point for your application.
Move everything else to a separate class.